### PR TITLE
feat(skeleton): add skeleton component review

### PR DESCRIPTION
**Pull request by lucasaarcoverde**
#### What is the purpose of this pull request?

As the title says.

[Design specs](https://www.figma.com/file/sepBhrHViUHyrTnktkuHaW/Admin-UI-%E2%80%93-Components?node-id=6717%3A424963)

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Quality improvement (tests or refactors)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Trivial change (small fix or feature that doesn't impact functionalities)
- [ ] Requires change to documentation, which has been updated accordingly

### CODE REVIEW

**Review by matheusps — APPROVED** [review]
LGTM

File: packages/admin-ui/src/skeleton/skeleton.style.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  },
+  '100%': {
+    transform: 'translateX(100%)',
+  },
+})
+
+export const baseline = style({
+  display: 'block',
+  position: 'relative',
+  width: 'full',
```
Comment (Lenderson) [inline packages/admin-ui/src/skeleton/skeleton.style.ts:29]: Do we should use as `$full`?

File: packages/admin-ui/src/skeleton/skeleton.style.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { keyframes, style, styleVariants } from '@vtex/admin-ui-core'
+
+export const variants = styleVariants({
+  shape: {
+    rect: {
+      borderRadius: 'default',
```
Comment (Lenderson) [inline packages/admin-ui/src/skeleton/skeleton.style.ts:6]: `$default`

File: packages/admin-ui/src/skeleton/skeleton.style.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    },
+    circle: {
+      borderRadius: 'circle',
```
Comment (Lenderson) [inline packages/admin-ui/src/skeleton/skeleton.style.ts:9]: `$circle`

**Review by Lenderson — APPROVED** [review]
LGTM, just some simple suggestions

File: packages/admin-ui/src/skeleton/skeleton.style.ts (hunk 1 above)
Comment (lucasaarcoverde) [reply packages/admin-ui/src/skeleton/skeleton.style.ts:29]: I'll change to `100%` instead since we don't have plans to define size tokens for now

**Pull request merged**